app/models/admin/FeatureFlags.scala

```diff
@@ -144,3 +140,31 @@ case object BreathingSpaceIndicatorToggle extends FeatureFlagName {
     "Enable/disable calls to Breathing space"
   )
 }
+
+case object AddressJourneyEnforcerToggle extends FeatureFlagName {
+  override val name: String                = "address-journey-enforcer-toggle"
+  override val description: Option[String] = Some(
+    "Enable/disable call to for address journey"
```

Comment:
Is this the change of address or there is more?
"Enable/disable the change of address journey"

Reply:
This journey is to allow the user to edit the address or to show that address is changed.